scripts/packages/packager/signed-entrypoint.sh

```diff
@@ -4,7 +4,12 @@ set -e
 set -x
 set -euxo pipefail
 
-FREEBSD_DISTROS="FreeBSD:12:amd64 FreeBSD:13:amd64"
+case "$(uname -m)" in
```

Comment:
Can we undo the changes here just for the signed-entrypoint.sh? 
The changes for the local-entrypoint.sh are fine but I think if are to support FreeBSD aarch64 packages in the future we will need to make updates here https://github.com/nginx/agent/blob/main/Makefile.packaging#L165 to the Makefile.packaging to run the docker container twice. Once for aarch64 and once for amd64. Passing in a different binary build each time.

Reply:
@dhurley sure, will do.

(I'm struggling to understand how it would help though given that goreleaser/NFPM is able to produce multiarch binaries on any host arch, and you can reuse the same binaries in "signed" builder in exactly the same way as in "local" builder.)

Reply:
@dhurley latest commit (https://github.com/nginx/agent/pull/350/commits/5e8c0917af5d27bf2b6b4b7f0bfc994b02e9cc41) reverted all functional changes in signed-entrypoint.sh back to its original state - isn't that that you requested?

Reply:
oh yes you're correct sorry I missed that. Looks good now.